content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,351 @@
+* ec2:DescribeSubnets
+* ec2:DescribeVpcs
+* iam:AttachRolePolicy
+* iam:CreatePolicy
+* iam:CreateRole
+* iam:PassRole
+* iam:PutRolePolicy
+* iam:UpdateAssumeRolePolicy
+* kms:Decrypt
+* lambda:AddPermission
+* lambda:CreateFunction
+* lambda:GetCodeSigningConfig
+* lambda:GetFunction
+* lambda:GetFunctionCodeSigningConfig
+* lambda:InvokeFunction
+* lambda:PutFunctionConcurrency
+* logs:CreateLogGroup
+* logs:DescribeLogGroups
+* logs:PutRetentionPolicy
+* s3:CreateBucket
+* s3:GetObject
+* s3:GetObjectVersion
+* secretsmanager:CreateSecret
+* secretsmanager:GetSecretValue
+* secretsmanager:PutSecretValue
+* serverless:CreateCloudFormationTemplate
+
+2. A Datadog account and [API key][8]. If you need a Datadog account, [sign up for a free trial][9].
+
+## Setup
+
+1. You will need to provide a valid Datadog [API Key][10] to the CloudFormation template. You can paste the API key directly into the CloudFront template, or you can store the value in [Secrets Manager][11] and reference the ARN of the secret. First, copy this value to your clipboard.
+
+2. From the AWS tile on the [Integrations page][12] in your Datadog account, select the CloudFormation Template option:
+{{< img src="getting_started/integrations/integration-tile-setup.png" alt="An image from the Datadog AWS integration tile showing the options for establishing the integration. The Role Delegation tab is highlighted.">}}
+
+3. This opens the AWS Console and loads the CloudFormation stack. Paste the API key into the `DdApiKey` field, or paste the Secret ARN into the `DdApiKeySecretArn` if using Secrets Manager:
+{{< img src="getting_started/integrations/cloudformation-options-1.png" alt="An image from the AWS CloudFormation create-stack page that shows the Stack name as datadog, IAMRoleName as DatadogIntegrationRole, ExternalId as an obfuscated value ending in be46, DdApiKey as an obfuscated value.">}}
+
+4. Ensure that the the correct [Datadog site][13] is used for the `DdSite` parameter to match the region of your Datadog account. Check the required boxes from AWS and click `Create stack`:
+{{< img src="getting_started/integrations/cloudformation-options-2.png" alt="An image from the AWS CloudFormation create-stack page that shows the Advanced options of creating the Datadog stack. DdAWSAccountId parameter is filled in with 464622532012, DdForwarderName parameter is filled in with DatadogForwarder, and InstallDatadogPolicyMacro is set as true. Below these parameters is a Capabilities section with two checkboxes, both of which are checked. The first checkbox states 'I acknowledge that AWS CloudFormation might create IAM resources with custom names.' The second checkbox states 'I acknowledge that AWS CloudFormation might require the following capability: CAPABILITY_AUTO_EXPAND'.">}}
+
+This begins the creation process for the Datadog stack along with three nested stacks. This could take several minutes; ensure that the stack successfully creates before proceeding:
```

Comment:
```suggestion
This begins the creation process for the Datadog stack along with three nested stacks. This could take several minutes; ensure that the stack is successfully created before proceeding:
```